docs/blog/2023-11-09-release-v2.0.0/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: It's time for some changes - Woodpecker 2.0.0
+description: Introducing Woodpecker 2.0.0 with > 300 changes
+slug: release-v200
+draft: true
+authors:
+  - name: Anbraten
+    title: Maintainer of Woodpecker
+    url: https://github.com/anbraten
+    image_url: https://github.com/anbraten.png
+tags: [release, stable]
+hide_table_of_contents: false
+---
+
+We are proud to present you Woodpecker v2.0.0 with more than 300 changes from our great community. This release includes a lot of new features, improvements and some breaking changes which most of you probably already tested using the `next` tag.
+
+<!--truncate-->
+
+## Breaking changes
```

Comment:
Should this be sorted a bit? I guess a least reading the "most important" breaking changes first could be helpful instead of starting with an int64 change?

1. platform filter
2. update docker
3. removal of API paths
4. removal of ssh backend
5. etc?

Reply:
Yes I also planned to write (copy from the migrations notes) a bit about how this effects the user and how they need to change things.